SnipperAPI/Interfaces/IEncrypt.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace SnipperAPI.interfaces
+{
+    public interface IEncrypt
+    {
+        string Decrypt(byte[] content);
+        byte[] Encrypt(string content);
+    }
+}
```

Comment:
Github has a real issue with new lines at the end of files, wish I could tell you why but when using github its important to add an empty line at the bottom of each file or you will see that small validation warning in the git diff
```suggestion
}

```